### PR TITLE
added docs for mercurius-explain-graphiql-plugin integration

### PR DESCRIPTION
**Pull request by marco-ippolito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,3 +125,26 @@ app.register(explain, {
...
+  schema,
+  resolvers,
+  graphiql: {
+    enabled: true,
+    plugins: [
+      {
+        name: 'mercurius-explain-graphiql-plugin',
+        props: { title: 'Mercurius explain graphiql plugin' },
+        umdUrl:
+          'https://unpkg.com/mercurius-explain-graphiql-plugin/dist/umd/index.js',
```
Comment (marco-ippolito) [inline README.md:144]: maybe it is possibile to wrap this into a function that this library exposes

Comment (simoneb) [reply README.md:144]: yep it would be good addition to simplify the usage of the plugin!

File: index.js [hunk 2]
```diff
@@ -43,3 +43,13 @@ async function isEnabled(options, { schema, source, context }) {
     return false
   }
 }
+
+export function explainGraphiQLPlugin() {
+  return {
+    name: 'mercurius-explain-graphiql-plugin',
```
Comment (codeflyer) [inline index.js:49]: This is not a valid name. It should be used the same used in the `rollup` export. (`mercuriusExplain`)


```
      {
        file: packageJson.umd,
        format: 'umd',
        sourcemap: true,
        name: 'mercuriusExplain'
      }
```

File: index.js [hunk 3]
```diff
@@ -43,3 +43,13 @@ async function isEnabled(options, { schema, source, context }) {
...
+    props: { title: 'mercurius explain graphiql plugin' },
```
Comment (codeflyer) [inline index.js:50]: The Plugin do not take the `title` as prop.

You can remove the whole line.

**Review by codeflyer — APPROVED** [review]

**Pull request merged**